rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -47,7 +48,8 @@ def ppo_surrogate_loss(
 
     # RNN case: Mask away 0-padded chunks at end of time axis.
     if state:
-        max_seq_len = tf.reduce_max(train_batch["seq_lens"])
+        B = tf.shape(train_batch["seq_lens"])[0]
```

Comment:
This was not safe for attention nets, where the max seq len must be kept (attention nets are not flexible wrt the seq-len as RNNs are).